source/fem-dem/vans_assemblers.cc

```diff
@@ -1589,6 +1589,62 @@ GLSVansAssemblerMagnus<dim>::calculate_particle_fluid_interactions(
 template class GLSVansAssemblerMagnus<2>;
 template class GLSVansAssemblerMagnus<3>;
 
+template <int dim>
+void
+GLSVansAssemblerOneWayViscousTorque<dim>::calculate_particle_fluid_interactions(
+  NavierStokesScratchData<dim> &scratch_data)
+{
+  // particle_number is an increment that goes from 0 to n_particles_in_cell.
+  // It is incremented at the end of the loop over particles
+
+  // Physical Properties
+  Assert(
+    !scratch_data.properties_manager.is_non_newtonian(),
+    RequiresConstantViscosity(
+      "GLSVansAssemblerOneWayViscousTorque<dim>::calculate_particle_fluid_interactions"));
+
+  Assert(
+    scratch_data.properties_manager.density_is_constant(),
+    RequiresConstantDensity(
+      "GLSVansAssemblerOneWayViscousTorque<dim>::calculate_particle_fluid_interactions"));
+
+  double viscous_torque = 0.0;
+
+  const double density = scratch_data.properties_manager.get_density_scale();
+  const double viscosity =
+    scratch_data.properties_manager.get_viscosity_scale();
+
+  auto &velocity_curls_3d =
+    scratch_data.fluid_velocity_curls_at_particle_location_3d;
+
+  const auto pic = scratch_data.pic;
+
+  unsigned int particle_number = 0;
+
+  // Loop over particles in cell
+  for (auto &particle : pic)
+    {
+      auto particle_properties = particle.get_properties();
+
+      for (unsigned int d = 0; d < dim; d++)
+        {
+          // Calculate viscous torque
+          viscous_torque =
+            8.0 * M_PI * viscosity * density *
+            Utilities::fixed_power<3, double>(
+              particle_properties[DEM::PropertiesIndex::dp] * 0.5) *
+            (velocity_curls_3d[particle_number][d] -
+             particle_properties[DEM::PropertiesIndex::omega_x + d]);
```

Comment:
I think there is an issue here. The real formula is 0.5 * rot(velocity). Why is it velocity_curls? Why not just call it vorticity?